### PR TITLE
adding french translation

### PR DESCRIPTION
**Pull request by tconstans**
Hello,

Just created a quick (and probably) dirty french translation of ui.h file.

Best regards

### CODE REVIEW

**Comment by allinurl** [comment]
Thanks very much for this PR! I expect to release a minor version that addresses issues #422 and #433, soon, so for now I'll hold on this PR. However, once v1.0.2 is deployed, I can merge this so it can go out in  v1.0.3.

File: src/ui_fr.h [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+#define MAX_HEIGHT_FOOTER 1     /* height of the footer window */
+#define MAX_HEIGHT_HEADER 7     /* height of the header window */
+#define OVERALL_NUM_COLS  4     /* number of columns on the overall stats win */
+
+/* Overall Stats Labels */
+#define T_DASH       "Tableau de bord"
+#define T_HEAD       "Requètes analysées, vue d'ensemble"
+
+#define T_DATETIME   "Date/Heure"
+#define T_REQUESTS   "Nb total de requètes"
```
Comment (greg0ire) [inline src/ui_fr.h]: `:%s/equètes/equêtes/`

File: src/ui_fr.h [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+#define T_GEN_TIME   "Temps de calcul"
+#define T_FAILED     "Requètes échouées"
+#define T_VALID      "Requètes valides"
+#define T_UNIQUE_VIS "Visiteurs uniques"
+#define T_UNIQUE_FIL "Fichiers uniques"
+#define T_EXCLUDE_IP "Excl. IP Hits"
+#define T_REFERRER   "Origine"
+#define T_UNIQUE404  "Unique 404"
```
Comment (greg0ire) [inline src/ui_fr.h]: "404 uniques" ?

File: src/ui_fr.h [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+#define REQUE_ID    "requests"
+#define REQUE_LABEL "Requètes"
+
+#define STATI_HEAD  "Requètes statiques"
+#define STATI_DESC  "Top des requètes statiques trié par hits [, avgts, cumts, maxts, mthd, proto]"
+#define STATI_ID    "static_requests"
+#define STATI_LABEL "Requètes Statiques"
+
+#define VTIME_HEAD  "Distribution temporelle"
+#define VTIME_DESC  "Données triée par heure [, avgts, cumts, maxts]"
```
Comment (greg0ire) [inline src/ui_fr.h]: triées

File: src/ui_fr.h [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+#define INCLUDE_BOTS " - robots inclus"
+
+/* Module Labels and Descriptions */
+#define VISIT_HEAD  "Visiteurs uniques par jour"
+#define VISIT_DESC  "Les hits depuis la même IP, date et user-agent comptent comme visite unique"
+#define VISIT_ID    "visitors"
+#define VISIT_LABEL "Visiteurs"
+
+#define REQUE_HEAD  "Fichiers demandés (URLs)"
+#define REQUE_DESC  "Top des requètes triés par hits [, avgts, cumts, maxts, mthd, proto]"
```
Comment (greg0ire) [inline src/ui_fr.h]: "trié" or "triée", but not "triés"

**Comment by tconstans** [comment]
I agree w/ Greg's corrections

File: src/ui_fr.h (hunk 4 above)
Comment (ahpnils) [inline src/ui_fr.h]: If I may : "requêtes" instead of "requètes".

File: src/ui_fr.h (hunk 3 above)
Comment (ahpnils) [inline src/ui_fr.h]: "requêtes" instead of "requètes".

**Comment by haricot** [comment]
@ahpnils We use gettext to translate instead of the raw translation. #701 #712 
You can see if everything suits you on the file [fr.po](https://github.com/allinurl/goaccess/blob/master/po/fr.po)
To have translations, we have to compile it from the git because it is unreleased at the moment, all fields are not ready to be translated.

**Comment by allinurl** [comment]
@haricot btw, soon I'll be pushing the changes that fix the terminal dashboard overflow issue. I'll keep you posted.

**Comment by allinurl** [comment]
i18 has been implemented and the next release will be shipped with a French translation. Thanks.

**Pull request closed without merging**